Keyboards/KeyboardsBase/KeyboardKeys.swift

```diff
@@ -51,7 +51,11 @@ class KeyboardKey: UIButton {
       self.layer.setValue(true, forKey: "isSpecial")
```

Comment:
Just an observation on the changes in this file -

These are interesting, hmm.. :thinking: Makes sense that some keys should be ignored when it comes to upper/lower-casing. Just wondering if there could be a scenario where a key would have upper/lower-casing in one language but not in another. In that case, the logic here would depend on the target language. I'm thinking this would definitely be an edge-case, and I don't even know if it could happen :laughing: , but just a thought I had.

Reply:
Thoughts of edge cases are much appreciated! There are all kinds of little things like that in Scribe 😊 Specifically `languagesWithCapitalizedNouns`, which as of now is `["German"]` 😄 We try to handle them as they come up :)

Reply:
Oh.. the challenges of working with multiple human languages :laughing:

Reply:
Welcome to Scribe! 🤣😊